.gitignore

```diff
@@ -10,3 +10,4 @@ coverage.xml
 pep8.txt
 .tox/
 .coverage
+.idea/
```

Comment:
Rather don't add this to all project gitignores, rather add to your global gitignore